lib/takeaway.rb

```diff
@@ -0,0 +1,73 @@
+require 'twilio-ruby'
+
```

Comment:
Would be good to have tests!
Run rubocop to pick up 33 suggestions for improving the code.